console_socket.c

```diff
@@ -36,7 +36,7 @@ static int console_socket_init(struct ge *ge, void *ctx)
 }
 
 
-static int console_socket_check(struct ge *ge, void *ctx)
+int console_socket_check(struct ge *ge, void *ctx)
```

Comment:
I think using some console_socket functions is a good way to interact with the console when ge isn't running.